addons/mail/models/mail_link_preview.py

```diff
@@ -29,8 +26,8 @@ class MailLinkPreview(models.Model):
     image_mimetype = fields.Char('Image MIME type')
     create_date = fields.Datetime(index=True)
 
-    def _bus_channel(self):
-        return self.message_id._bus_channel()
+    def init(self):
+        self.env.cr.execute("CREATE UNIQUE INDEX IF NOT EXISTS mail_link_preview_source_url ON %s (source_url)" % self._table)
```

Comment:
Seems in master we have ``models.UniqueIndex`` that allows to define unique indexes ... did not play with it, but maybe worth migrating to the new API ?

Reply:
Look like it's working, so I guess it's better to use `UniqueIndex`

Reply:
Marked as solved, maybe not pushed ?

Reply:
Wrong comment instead, I updated the PR